jest.config.ts

```diff
@@ -24,4 +24,11 @@ export default {
   ],
   modulePaths: ['<rootDir>'],
   moduleDirectories: ['node_modules'],
+  globals: {
```

Comment:
Seems like the library got TS-Errors in their `index.ts` not sure how to ignore these in jest (tests fail) in a better way. You can comment out this and run `UserVocabularyProcessScreen.spec` to check error

Reply:
Disabling typechecking is currently not possible for importet files in typescript.
See https://github.com/Microsoft/TypeScript/wiki/FAQ#why-is-a-file-in-the-exclude-list-still-picked-up-by-the-compiler
Best would probably be to open an pr in the repo and wait until they resolve this...

Reply:
and so i should keep this pr open until fixed? This can take months... I can open an issue and mention that in the `jest.config.js`

Reply:
fixed this with a path. gonna create a issue in the library for that